src/screens/FeedScreen/Header.js

```diff
@@ -233,7 +233,7 @@ const _renderAnonimity = ({
                       refSheet.current.open();
                     }}>
                     <View style={{zIndex: 1000}}>
-                      <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
+                      <ElipsisIcon width={4} height={14} fill={COLORS.gray400} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ElipsisIcon` has been changed from `COLORS.blackgrey` to `COLORS.gray400`. Ensure that this color change aligns with the overall theme and accessibility standards of your application.
```diff
- <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
+ <ElipsisIcon width={4} height={14} fill={COLORS.gray400} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->